src/drt/src/frDesign.h

```diff
@@ -89,6 +93,7 @@ class frDesign
   std::vector<std::unique_ptr<frMaster>> masters_;
   std::unique_ptr<frTechObject> tech_;
   std::unique_ptr<frRegionQuery> rq_;
+  std::vector<std::string> user_defined_vias_;
```

Comment:
I would call them user_selected_vias as the user isn't defining the vias themselves but merely choosing them.